### ISSUE TITLE
A promise was created in a handler but was not returned from it.

### ISSUE THREAD

**Issue by girard-r** [Microsoft/TypeScript#26567]
Hi,

  I am using typescript with nodemon for development purposes.
Each time i launch nodemon, everything is working as expected but i get this warning: 

```
(node:10105) Warning: a promise was created in a handler at anonymous function) [as .ts] (.../node_modules/ts-node/src/index.ts:442:12 but was not returned from it, see http://goo.gl/rRqMUw
    at Function.Promise.attempt.Promise.try (.../node_modules/bluebird/js/release/method.js:29:9)
```

I do not have the issue if i compile and serve the project.

**Comment by AnyhowStep** [comment]
This isn't a TS problem.
The solution is in the warning message,

http://goo.gl/rRqMUw

**Comment by RyanCavanaugh** [comment]
It looks like this is a question rather than a bug report. This issue tracker is for tracking bugs and active work on TypeScript *itself*, rather than a general forum for programmers using TypeScript to get help or ask questions.

You can ask questions on sites like [Stack Overflow](https://stackoverflow.com/questions/tagged/typescript). We are not able to provide one-on-one support on the issue tracker. Please read the issue template carefully - it has important information on what kinds of reports can be acted on here, as well as links to useful TypeScript resources. Thanks!
